packages/console/src/components/PlanUsage/PlanUsageCard/index.tsx

```diff
@@ -32,6 +33,7 @@ function PlanUsageCard({
   usageKey,
   titleKey,
   tooltipKey,
+  isUsageTipHidden,
```

Comment:
Can we use `isEnterprisePlan` inside this component? So you don't need to pass the param

Reply:
Was using `isEnterprisePlan` flag in the component, but since we will have different display logic for all quota usages, better to keep the flag simple and we can configure them from outer scope.